pkg/servicemesh/apis/servicemesh/v1/doc.go

```diff
@@ -1,3 +1,17 @@
+// Copyright Istio Authors
```

Comment:
@knrc who owns the copyright for code that we write in our fork of Istio? Should this be copyright Istio or copyright Red Hat? Or even copyright Maistra?

Reply:
This does need to change, the answer is that we should not be using "Copyright Istio Authors" unless it's part of the upstream